components/AddContentProviderForm/AddContentProviderForm.tsx

```diff
@@ -0,0 +1,64 @@
+import type { ChangeEventHandler, FormEventHandler } from 'react';
+import { useCallback, useState } from 'react';
+
+import { Button } from '../Button/Button';
+
+import styles from './addContentProviderForm.module.scss';
+import { useFormReducer } from './utils';
+
+export const AddContentProviderForm = () => {
```

Comment:
`Provider` to raczej słowo zastrzeżone dla Reactowych Providerów. Poza tym nie brzmi to zbyt dobrze – po angielsku nigdy byś nie powiedział "Content Provider" o człowieku :P